tai64/CHANGELOG.md

```diff
@@ -4,6 +4,22 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## 5.0.0 (2024-10-29)
+### Changed
+- bump zeroize from 1.4.2 to 1.4.3 ([#187])
+- Lint improvements ([#824])
+- bump zeroize from 1.5.7 to 1.6.0 ([#956])
+- bump zeroize from 1.7.0 to 1.8.1 ([#1419])
```

Comment:
I don't think we need 3 entries in the changelog for bumping `zeroize`. Also, are the bumps even necessary? I don't think anything changed.

Lint improvements is likewise not a user facing/relevant change.